.vscode/settings.json

```diff
@@ -19,6 +19,7 @@
         "utility": "c",
         "ranges": "c"
     },
+    "editor.formatOnSave": false,
```

Comment:
Could this change be reverted, please?

Reply:
Yes, sure it can.
But then, how do you configure VSCode to not screw up your layout when you save?

Reply:
Well, saving the setting to the account/user, rather than workspace would do it.  

But I think that we should probably ignore the entire .vscode folder. 

But in the meanwhile: 
```sh
git update-index --skip-worktree .vscode/settings.json
```

Reply:
> Well, saving the setting to the account/user, rather than workspace would do it.

The thing is that, by default, I use format on save. Doesn't VSCode totally ruin your layouts when you save?

Reply:
```suggestion
```

Reply:
> The thing is that, by default, I use format on save. Doesn't VSCode totally ruin your layouts when you save?

Yes, and no.  It uses clang-format to reformat the files.  
You can use :
```c
// clang-format off
stuff to no format
// clang-format on
```

This prevents it from screwing up certain blocks.